pkg/limits/ingest_limits_test.go

```diff
@@ -378,6 +431,10 @@ func TestIngestLimits_GetStreamUsage_Concurrent(t *testing.T) {
 				StreamHashes: []uint64{1, 2, 3, 4, 5},
 			}
 
+			// Assign the Partition IDs.
+			partitions := map[string][]int32{"tenant1": {0}}
```

Comment:
Can we calculate this from the test case instead of hardcode it for all tests? The tenant ID does not have to been `tenant1` as it's configurable on a per-test basis.